nav2_behavior_tree/plugins/action/compute_path_to_pose_action.cpp

```diff
@@ -40,20 +40,23 @@ void ComputePathToPoseAction::on_tick()
 BT::NodeStatus ComputePathToPoseAction::on_success()
 {
   setOutput("path", result_.result->path);
+  setOutput("global_planner_result_code", result_.result->error_code);
   return BT::NodeStatus::SUCCESS;
 }
 
 BT::NodeStatus ComputePathToPoseAction::on_aborted()
 {
   nav_msgs::msg::Path empty_path;
   setOutput("path", empty_path);
+  setOutput("global_planner_error_code", result_.result->error_code);
   return BT::NodeStatus::FAILURE;
 }
 
 BT::NodeStatus ComputePathToPoseAction::on_cancelled()
 {
   nav_msgs::msg::Path empty_path;
   setOutput("path", empty_path);
+  setOutput("global_planner_error_code", result_.result->error_code);
```

Comment:
If it was aborted, there would be no error code (or I suppose the error code would be that we canceled it, not really an error though) so set to 0?